Wasm-container/README.md

```diff
@@ -0,0 +1,509 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-orange.svg)](https://opensource.org/licenses/Apache-2.0)
+
+# Vertica Web Assembly UDx experimental container
+
+[Vertica](https://www.vertica.com/) is a massively scalable analytics data warehouse that stores your data and performs analytics on it all in one place.
+
+This repository creates a container image that makes it reasonably easy to experiment with developing User-Defined Extensions (UDxes) for Vertica using WebAssembly.
+
+Vertica has long supported UDxes in a number of languages:
+
+- C++
+- Java
+- Python
+- R
+
+This container creates an environment for creating UDxes in Rust (and,
+as a side-effect, plain-old C).
+
+Java, Python and R UDxes are loaded into auxiliary programs
+implementing the appropriate runtime.  C++ UDxes may be loaded into a
+separate program (known as "fenced"), or may be linked directly into
+the Vertica binary (known as "unfenced", the implication being there's
+nothing between your UDx code and the innards of Vertica).  All Java, Python, and R UDxes are fenced.
+
+Invoking unfenced functions is done by calling a function with
+arguments.  Invoking fenced functions is done through remote procedure
+calls.
```

Comment:
Because markdown can be a little precious about spacing, check that this renders ok (no stray linebreaks in the middle of the paragraph).